src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+@import "../utils/variables.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  line-height: 14px;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+
+  &__image {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    height: 134px;
+    width: 160px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    margin: 0;
+    padding-bottom: 4px;
+
+    color: #060b35;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin: 0;
+    padding-bottom: 16px;
+  }
+
+  &__tag {
+    padding: 0;
+
+    line-height: 18px;
+    font-size: 12px;
+  }
+
+  &__cost {
+    padding: 0;
+
+    line-height: 18px;
+    font-size: 16px;
+    font-weight: 700;
+    color: #060b35;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+
+    padding-bottom: 24px;
+  }
+
+  &__reviews {
+    margin: 0;
+    display: flex;
+    align-items: center;
+
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin: 0;
+    padding-bottom: 16px;
+  }
+
+  &__button_link {
+    display: block;
+
+    margin: 0;
+    padding: 11px 0;
+
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    background-color: #00acdc;
+
+    text-align: center;
+    text-decoration: none;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+    text-transform: uppercase;
+
+    transition: color 0.3s, background-color 0.3s;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions